lumicks/pylake/correlated_stack.py

```diff
@@ -300,6 +299,7 @@ def crop_and_rotate(self, frame=0, channel="rgb", show_title=True, **kwargs):
         """Open a widget to interactively edit the image stack.
 
         Actions include:
+
```

Comment:
I think it would be nice here to define our own `Actions` header to get some nice formatting. You can use `napoleon_custom_settings` in `conf.py`

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/sphinxcontrib.napoleon.html#sphinxcontrib.napoleon.Config

Reply:
Ah yes, great idea, that'd be cool. I can add that later this afternoon.

Reply:
Added!